content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -26,7 +26,7 @@ mechanism that allows different cloud providers to integrate their platforms wit
 ![Kubernetes components](/images/docs/components-of-kubernetes.svg)
 
 The cloud controller manager runs in the control plane as a replicated set of processes
-(usually, these are containers in Pods). Each cloud-controller-manager implements
```

Comment:
The new wording seems a bit more likely to confuse readers - to me it seems harder to parse into phrases.

It could be:
The (cloud controller manager) runs (in (the control plane) as a (replicated set) of processes) such as (containers in Pods).
or
The (cloud controller manager) runs in (the control plane) as (a replicated set of (processes (such as (containers in Pods)))).